### DISCUSSION TITLE
presidio-analyzer : how to recieve "analyzer_results" answer

### DISCUSSION THREAD

**Discussion by nir106** [microsoft/presidio#695] [Q&A] [answered]
I am trying to run a project in C# (the containers are up and running :-) )
Using the presidio-analyzer and the presidio-anonymizer.

The result I am getting from the Analyze call contains  "analysis_explanation" form and I am trying to figure how can I receive the "analyzer_results" answer. 
i converted the curl to http post in reqbin.com site
curl -X POST http://localhost:5001/analyze -H "Content-type: application/json" --data  "{ \"text\": \"John Smith drivers license is AC432223\", \"language\" : \"en\"}"

hope I presented my problem clearly

thank you for your help in advance,

**Comment by navalev** [comment, marked as answer]
Hi @nir106, the result of this request should be:
```json
[
    {
        "analysis_explanation": null,
        "end": 10,
        "entity_type": "PERSON",
        "score": 0.85,
        "start": 0
    },
    {
        "analysis_explanation": null,
        "end": 38,
        "entity_type": "US_DRIVER_LICENSE",
        "score": 0.6499999999999999,
        "start": 30
    }
]
```
There are several postman collections with sample request under our documentation https://microsoft.github.io/presidio/samples/docker/#postman-collection 

what is the request result you are getting?
